access-control-app/int-test/cmr/access_control/int_test/access_control_group_crud_test.clj

```diff
@@ -103,8 +103,10 @@
 
     (testing "Create group with fields at maximum length"
       (let [group (into {} (for [[field max-length] field-maxes]
-                             [field (string-of-length max-length)]))]
-        (is (= 200 (:status (u/create-group (e/login (u/conn-context) "user1") group)))))))
+                             [field (string-of-length max-length)]))
+            ;; FIXME understand why having this outside of a binding causes JVM corruption error
```

Comment:
This one makes me a little nervous. I'd love to understand why we see the stack corruption error if trying to make the call to create-group outside the binding.

I can't reproduce this problem outside of two tests, but it boils down to:

JVM corruption:
```
(let [group {:group-stuff "ABC"}]
  (create/group group)
```
Works fine: 🤔 
```
(let [group {:group-stuff "ABC"}]
      _ (create/group group)
```